packages/core/test/specHelper.ts

```diff
@@ -151,15 +138,91 @@ export function stubFetch(): FetchStubManager {
   }
 }
 
-export interface ResponseStub extends Partial<Response> {
+export interface ResponseStubOptions {
+  status?: number
+  method?: string
+  type?: ResponseType
   responseText?: string
   responseTextError?: Error
 }
+function todo(): never {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** since it is not really something that we have to do‏
```suggestion
function notYetImplemented(): never {
```